modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/08_search/page-definition.json

```
@@ -0,0 +1,330 @@
+										"type": "Widget"
+									},
+									{
+										"definition": {
+											"fragmentStyle": {
+												"backgroundColor": "colorNeutral0",
+												"borderColor": "colorNeutral3",
+												"borderRadius": "borderRadiusSm",
+												"borderWidth": "1",
+												"marginBottom": "4",
+												"marginLeft": "4",
+												"marginRight": "4",
+												"marginTop": "4"
+											},
+											"widgetInstance": {
+												"widgetConfig": {
+												},
+												"widgetName": "com_liferay_portal_search_web_tag_facet_portlet_TagFacetPortlet"
+											}
+										},
+										"type": "Widget"
+									},
+									{
+										"definition": {
+											"fragmentStyle": {
+												"backgroundColor": "colorNeutral0",
+												"borderColor": "colorNeutral3",
+												"borderRadius": "borderRadiusSm",
+												"borderWidth": "1",
+												"marginBottom": "4",
+												"marginLeft": "4",
+												"marginRight": "4",
+												"marginTop": "4"
+											},
+											"widgetInstance": {
+												"widgetConfig": {
+												},
+												"widgetName": "com_liferay_portal_search_web_type_facet_portlet_TypeFacetPortlet"
+											}
+										},
+										"type": "Widget"
+									},
+									{
+										"definition": {
+											"fragmentStyle": {
+												"backgroundColor": "colorNeutral0",
+												"borderColor": "colorNeutral3",
+												"borderRadius": "borderRadiusSm",
+												"borderWidth": "1",
+												"marginBottom": "4",
+												"marginLeft": "4",
+												"marginRight": "4",
+												"marginTop": "4"
+											},
+											"widgetInstance": {
+												"widgetConfig": {
+												},
+												"widgetName": "com_liferay_portal_search_web_modified_facet_portlet_ModifiedFacetPortlet"
+											}
+										},
+										"type": "Widget"
+									}
+								],
+								"type": "Section"
+							},
+							{
+								"definition": {
+									"widgetInstance": {
+										"widgetConfig": {
+										},
+										"widgetName": "com_liferay_portal_search_web_search_results_portlet_SearchResultsPortlet"
+									}
+								},
+								"type": "Widget"
+							}
+						],
+						"type": "Section"
+					},
+					{
+						"definition": {
+							"widgetInstance": {
+								"widgetConfig": {
+								},
+								"widgetName": "com_liferay_search_experiences_web_internal_blueprint_options_portlet_SXPBlueprintOptionsPortlet",
+								"widgetPermissions": [
+									{
+										"actionKeys": [
+											"PERMISSIONS",
+											"PREFERENCES",
+											"CONFIGURATION",
+											"VIEW",
+											"ADD_TO_PAGE"
+										],
+										"roleKey": "Owner"
+									},
+									{
+										"actionKeys": [
+										],
+										"roleKey": "Site Member"
+									},
+									{
+										"actionKeys": [
+											"VIEW"
+										],
+										"roleKey": "Guest"
+									}
```

```suggestion
									{
										"actionKeys": [],
										"roleKey": "Guest"
									}
```

Guest cannot have permissions